packages/aws-cdk-lib/aws-cloudfront/lib/function.ts

```diff
@@ -100,6 +100,12 @@ export interface FunctionAttributes {
    * The ARN of the function.
    */
   readonly functionArn: string;
+
+  /**
+   * The Runtime of the function.
+   */
+  readonly functionRuntime: string;
```

Comment:
```suggestion
  readonly functionRuntime?: string;
```

Build is failing with:

```
err  - IFACE aws-cdk-lib.aws_cloudfront.FunctionAttributes: newly required property 'functionRuntime' added: input to aws-cdk-lib.aws_cloudfront.Function.fromFunctionAttributes [strengthened:aws-cdk-lib.aws_cloudfront.FunctionAttributes]
```

Looks like we cannot require this prop to `fromFunctionAttributes` and instead will have to default to the original runtime, or throw an error if not provided.

Reply:
@watany-dev

Reply:
@kaizencc 
We decided to make it an arbitrary property. If not specified, the Default value, runtime1.0, is specified as before.